.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: "Found a Bug! 🐛"
+title: "🐛 [BUG] - <title>"
+description: "Use this template to report a bug. Thank you!"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks in advance for taking the time to file this issue!
+        We hope this simple template will improve issue resolution time and minimize the need for _context discovery_ discussions.
+        ______
+        **Note**: We encourage you to [submit an RFC](https://github.com/neon-bindings/rfcs) if you would like to propose a **design change** or a **new feature** for the Neon API or the Neon command-line tool.
+        ______
+
+        In addition to the `bug` label, consider adding others you might find relevant:
+        | Label | Description |
+        |:-----:|-------------|
+        | `CI` | Continuous Integration |
+        | `CLI` | Command Line Interface |
+        | `beginner friendly` | Great for those just starting out |
+        | `Performance` | Anything that affects performance |
+        | `linux` | Specific to Linux |
+        | `macos` | Specific to MacOS |
+        | `windows` | Specific to Windows |
+        | `android` | Specific to Android |
+        | `electron` | Specific to Electron apps |
+  - type: textarea
+    id: issue_summary
+    attributes:
+      label: Short Summary
+      description: Please provide a brief description of the issue
+      placeholder: "example: neon crashes when ..."
+    validations:
+      required: true
+  - type: textarea
+    id: issue_repro
+    attributes:
+      label: Steps to reproduce
+      description: "Help us reproduce this issue with some step-by-step instructions"
+      value: |
+        1. 
+        2. 
+        3. 
+        ...
+    validations:
+      required: true
+  - type: textarea
+    id: issue_fix
+    attributes:
+      label: Suggested fix
+      description: "If you have a proposed way to fix this bug, please summarize below. PRs work too :)"
+  - type: input
+    id: node_version
+    attributes:
+      label: "Node version: "
+      description: "Please specify the version of `node` you are running"
+      placeholder: "v18.10.0"
+  - type: input
+    id: npm_version
+    attributes:
+      label: "Node Package Manager version: "
```

Comment:
I think "Node" here is unnecessary, just "Package Manager version" is good.

Reply:
Perhaps just "Package Manager" to make space for alternate package managers (`yarn`, `pnpm`).

```yaml
    attributes:
      label: "Package Manage: "
      description: "Please specify the package manager and version you are running (`npm -v`)"
      placeholder: "npm 8.19.2"
```

Reply:
```yaml
    attributes:
      label: "Package Manager & version: "
      description: "Please specify the package manager your are using and its version"
      placeholder: "example: npm 8.19.2"
```
How's this?